tasks/eth/011-proto-ver+ownership-transfers/VALIDATION.md

```diff
@@ -27,16 +27,25 @@ To allow simulating the transaction bundle of setting the recommended versions a
 
 ### `0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935` (`ProtocolVersions`)
 
-There's a single state change for the `ProtocolVersions` contract:
 * Key: `0xe314dfc40f0025322aacc0ba8ef420b62fb3b702cf01e0cdf3d829117ac2ff1a`
 * Before: `0x0000000000000000000000000000000000000006000000000000000000000000`
 * After: `0x0000000000000000000000000000000000000007000000000000000000000000`
 * Meaning: This bumps the major version of the *recommended protocol version* from 6 to 7.
 
+* Key: `0x0000000000000000000000000000000000000000000000000000000000000033`
```

Comment:
so we wouldn't give a reference about why key `0x33` is for the owner slot?

Reply:
Added a change to clarify that Tenderly now displays this properly, so IMO we don't need to refer to the monorepo. 4623c78dfcb762774cbcf0559edf0a7c847a01b5